packages/config/src/layer2s/myria.ts

```diff
@@ -135,43 +122,6 @@ export const myria: Layer2 = {
       description:
         'Allowed to update the state. When the Operator is down the state cannot be updated.',
     },
-    {
-      name: 'Data Availability Committee',
```

Comment:
why are you deleting this? I thought we only remove mutlisig members, this one is actually pretty useful